Button/Button.module.less

```diff
@@ -288,6 +288,7 @@
 				background-color: @agate-button-focus-bg-color;
 				background-image: @agate-button-focus-bg-image;
 				border-color: @agate-button-focus-border-color;
+				background-origin: @agate-button-focus-background-origin;
```

Comment:
This code breaks the previous button look. Did you intend to do it.

Reply:
I think maybe this should just be simplified to `background-origin: @agate-button-background-origin;` and assigned to the regular resting button, rather than specifically on the `focus` state. It's not previously set, so having it only apply on focus seems incomplete.